pylint/checkers/utils.py

```diff
@@ -850,7 +850,7 @@ def uninferable_final_decorators(
         if isinstance(decorator, nodes.Attribute):
             try:
                 import_node = decorator.expr.lookup(decorator.expr.name)[1][0]
-            except AttributeError:
+            except (AttributeError, IndexError):
```

Comment:
I'm wondering if we should distinguish the two errors and try to handle the case where we can't do ``[1][0]``. I guess Daniel will have an opinion :)

Reply:
The IndexError occurs when `pytest` is not imported in the module; the tuple is empty in that case:
```python
> /Users/markbyrne/programming/pylint/pylint/checkers/utils.py(853)uninferable_final_decorators()
-> import_node = decorator.expr.lookup(decorator.expr.name)[1][0]
(Pdb) decorator.expr.lookup(decorator.expr.name)
(<Module.builtins l.0 at 0x108d82940>, ())
```

Reply:
I believe the logic is simpler now with 38cbeebdca1242b8a539cc85239808d90b12a571 & it more closely matches the `nodes.Name` case beneath it in the same function.

Reply:
No, need to re-add the try-except since the primer test fails without it.